projects/skia/build.sh

```diff
@@ -111,4 +124,10 @@ cp ./canvas_seed_corpus.zip $OUT/api_raster_n32_canvas_seed_corpus.zip
 
 cp out/Fuzz/api_null_canvas $OUT/api_null_canvas
 cp ./api_null_canvas.options $OUT/api_null_canvas.options
-cp ./canvas_seed_corpus.zip $OUT/api_null_canvas_seed_corpus.zip
\ No newline at end of file
+cp ./canvas_seed_corpus.zip $OUT/api_null_canvas_seed_corpus.zip
+
+# Remove unnecessary dependencies that aren't on runner containers.
```

Comment:
Here is where we make api_null_gl_canvas usable on OSS-Fuzz. 

I believe the NullGLTestContext inherits from GLTestContext which does depend on these libraries, but because of what NullGLTestContext overrides, it doesn't actually need these libraries. I may be wrong but I've run the fuzzers quite a bit and there didn't seem to be any issues.

Reply:
Can you add to the comment explaining how one knows to remove these things?  Was this trial and error or was it listed with a command?